autocomplete/urls.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
+
+from __future__ import unicode_literals
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/autocomplete/urls.py
+++ b/autocomplete/urls.py
@@ -1,6 +1,7 @@
 # -*- encoding: utf-8 -*-
 
 from __future__ import unicode_literals
+
 from autocomplete import views
 from django.conf.urls import url
 
```